idaes/models/properties/general_helmholtz/tests/test_stateblock.py

```diff
@@ -25,6 +25,7 @@
     StateVars,
 )
 from idaes.core import FlowsheetBlock
+from idaes.core.util.scaling import get_jacobian, jacobian_cond
```

Comment:
These imports aren't needed. I'm assuming you accidentally left them while looking into my former comment